{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/yaml/Exception/ParseException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Yaml\\Exception","location":[11,0,11,43]},{"kind":1,"name":"Symfony\\Component\\Yaml\\Exception\\ParseException","location":[18,0,140,1],"associated":[{"kind":1,"name":"Symfony\\Component\\Yaml\\Exception\\RuntimeException"}],"description":"Exception class thrown when an error occurs during parsing.","children":[{"kind":16,"name":"$parsedFile","modifiers":4,"location":[20,12,20,23],"scope":"Symfony\\Component\\Yaml\\Exception\\ParseException"},{"kind":16,"name":"$parsedLine","modifiers":4,"location":[21,12,21,23],"scope":"Symfony\\Component\\Yaml\\Exception\\ParseException"},{"kind":16,"name":"$snippet","modifiers":4,"location":[22,12,22,20],"scope":"Symfony\\Component\\Yaml\\Exception\\ParseException"},{"kind":16,"name":"$rawMessage","modifiers":4,"location":[23,12,23,23],"scope":"Symfony\\Component\\Yaml\\Exception\\ParseException"},{"kind":32,"name":"__construct","modifiers":1,"location":[34,4,44,5],"description":"Constructor.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","children":[{"kind":128,"name":"$message","location":[34,32,34,40],"description":"The error message","scope":"__construct","type":"string"},{"kind":128,"name":"$parsedLine","location":[34,42,34,58],"description":"The line where the error occurred","scope":"__construct","value":"-1","type":"int"},{"kind":128,"name":"$snippet","location":[34,60,34,75],"description":"The snippet of code near the problem","scope":"__construct","value":"null","type":"int"},{"kind":128,"name":"$parsedFile","location":[34,77,34,95],"description":"The file name where the error occurred","scope":"__construct","value":"null","type":"string"},{"kind":128,"name":"$previous","location":[34,97,34,124],"description":"The previous exception","scope":"__construct","value":"null","typeSource":1,"type":"Exception"}]},{"kind":32,"name":"getSnippet","modifiers":1,"location":[51,4,54,5],"description":"Gets the snippet of code near the error.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","type":"string"},{"kind":32,"name":"setSnippet","modifiers":1,"location":[61,4,66,5],"description":"Sets the snippet of code near the error.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","children":[{"kind":128,"name":"$snippet","location":[61,31,61,39],"description":"The code snippet","scope":"setSnippet","type":"string"}]},{"kind":32,"name":"getParsedFile","modifiers":1,"location":[75,4,78,5],"description":"Gets the filename where the error occurred.\n\nThis method returns null if a string is parsed.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","type":"string"},{"kind":32,"name":"setParsedFile","modifiers":1,"location":[85,4,90,5],"description":"Sets the filename where the error occurred.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","children":[{"kind":128,"name":"$parsedFile","location":[85,34,85,45],"description":"The filename","scope":"setParsedFile","type":"string"}]},{"kind":32,"name":"getParsedLine","modifiers":1,"location":[97,4,100,5],"description":"Gets the line where the error occurred.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","type":"int"},{"kind":32,"name":"setParsedLine","modifiers":1,"location":[107,4,112,5],"description":"Sets the line where the error occurred.","scope":"Symfony\\Component\\Yaml\\Exception\\ParseException","children":[{"kind":128,"name":"$parsedLine","location":[107,34,107,45],"description":"The file line","scope":"setParsedLine","type":"int"}]},{"kind":32,"name":"updateRepr","modifiers":4,"location":[114,4,139,5],"scope":"Symfony\\Component\\Yaml\\Exception\\ParseException"}]}]}}